_posts/2021-01-29-design-system-crazy.md

```diff
@@ -0,0 +1,137 @@
+nothing has actually changed inside.
+
+Nothing has changed because the leadership has not embraced that a Design System
+is a product, and nurturing a product requires a dedicated team.
+
+Is such a thing a pipe dream? It should not be, we are already used to this
+paradigm in our engineering departments. We have infrastructure or platform
+teams that cater to the other units, providing a solid platform to run their
+services. We have engineering productivity teams that help others with day to
+day tools (CI, testing environment, etc.). We have analysts and UX researchers
+to support the decision-making of other teams with bespoke data. They are all
+"in-house" services teams.
+
+Organizations willingly invest in all of those but can't form one for the Design
+System? What's the problem?
+
+Is it the idea of mixing designers and developers together as one unit? How is
+it different from what we do for feature teams, squads, swarms or whatever name
+you give to multidisciplinary teams, those that work on features?
+
+Is it an org chart issue? Were it's unclear whether such a team reports to the
+VP of Product or to the VP of Engineering? Why would that matter? Aren't they
+both after the same ultimate company goal?
+
+Is it because designers or developers are already understaffed? Well shoot,
+that's precisely why dedicating a team for it: a Design System is there to help
+you do more with less.
+
+Is it because nobody is here to lead this team? What about those who have
+managed to build the current Design System and do their day to day job *at the
+same time*? Aren't they leadership enough?
+
+Is it because you don't believe in investing in such low levels teams for the
+benefit of rest of the organization? Then stop talking about Design Systems,
+infrastructure teams, UX researchers and so on if you're against such an idea. 
+
+The point to all that questioning is: by wanting to produce a Design System
+without updating your organization to support it, you're making your designers
+and developers go crazy.
+
+They have to support their team projects on the roadmap *while* fighting with
+management for some time for Design System tasks. It's never the priority and
+the roadmap has never enough room for it. But they see its benefits, so they
+keep trying, and trying, and trying.
+
+They end up giving 100% for the Design System (because, you know, such a project
+is not an easy task) *while* they're asked to keep giving 100% on their
+regularly-scheduled features, knowing such a pace always ends poorly (yes,
+burnout). Or wry cynicism in the best of cases, go ask Benoit :)
+
+They have to face deep animosity from other colleagues *while* continuing to
+work with them on their features. Faced with such an unsanctioned, "bottom-up"
+project, some will have no issue undermining the Design System. If it's so
+important, a team would have been formed around it, right? Right.
```

Comment:
```suggestion
They are often put in acrimonious situations with other colleagues *while* continuing to
work with them on their features. These conflicts are unavoidable because the situation doesn't actually make sense to anyone. For the people invested in the Design System, it doesn't make sense not to fight for it. For those who aren't as invested, they are faced with an unsanctioned, "bottom-up"
project, and some will therefore have no issue undermining that project. If it were so
important, a team would have been formed around it, right? Right. 
```

Reply:
Je propose ça pour faire un poil plus court 

```suggestion
They are often put in acrimonious situations with other colleagues *while* continuing to
work with them on their features. This is unavoidable. The situation doesn't actually make sense to anyone. The people invested in the Design System will of course fight for it. Those who aren't as invested are faced with an unsanctioned, "bottom-up"
project, and some will therefore have no issue undermining that project. If it were so
important, a team would have been formed around it, right? Right. 
```